Documents/Help/Changelog.md

```diff
@@ -58,8 +58,7 @@
 * Added `KryptonLanguageManager` to the `KryptonManager` action list
 * Resolved [#1008](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1008), Krypton Save/Open file dialogs are not accessible from the toolbox
 * Implemented [#1007](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1007), A way to alter all of the strings in the toolkit to language specific strings
-* Implemented [#1006](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1006), Make converter strings localisable
-    - **Note:** Components that use these strings may need to be 'refreshed' once changes have been made
+* Implemented [#602](https://github.com/Krypton-Suite/Standard-Toolkit/issues/602), ToolStrip embedded into the non client area
```

Comment:
Why has this deleted 1006 ?

Reply:
It was removed from the code, as you can't change designer strings on the fly in VS